app/Makefile

```
@@ -26,15 +34,90 @@ else
 PY_RUN_CMD := poetry run
 endif
 
-run:
-	docker-compose up -d --build
+##################################################
+# API Build & Run
+##################################################
+
+build:
+	docker-compose build
+
+start:
+	docker-compose up --detach
+
+run-logs: start
+	docker-compose logs --follow --no-color $(APP_NAME) $(DECODE_LOG)
+
+
+init: build init-db
+
+clean-volumes:
+	docker-compose down --volumes
+
+stop:
+	docker-compose down
+
+##################################################
+# DB & migrations
+##################################################
+
+#########################
+# DB running / setup
+#########################
+
+# Docker starts the image for the DB but it's not quite
+# fully ready, so add a 5 second sleep so upgrade doesn't
+# fail because the DB hasn't started yet.
+init-db: start-db sleep-5 db-upgrade
+
+start-db:
+	docker-compose up --detach mock-api-db
```

Per my suggested changes in the `docker-compose.yml`, this would also need to change.
```suggestion
	docker-compose up --detach db
```